tasks/sync-granule/package.json

```diff
@@ -18,7 +18,7 @@
     "test-coverage": "nyc ava",
     "build": "rm -rf dist && mkdir dist && cp -R schemas dist/ && webpack --progress",
     "watch": "rm -rf dist && mkdir dist && cp -R schemas dist/ && webpack --progress -w",
-    "postinstall": "npm run build"
```

Comment:
Why is postinstall removed?

Reply:
This was breaking the `lerna ybootstrap` command.  `@cumulus/common` needs to be built in order to generate the `log.js` file, and that is done in the `prepublish` step.  That step is done _after_ the `postinstall` step, though.  This problem should be going away now that Alireza's Node 8 PR moves these builds out of `postinstall` and into `prepublish`.